tests/integration/tests/autocomplete.js

```diff
@@ -149,9 +149,9 @@ test('mouse navigation', async () => {
   expect(searchValue.trim()).toEqual(TypedSearch);
 
   await autocompleteHelper.clickResult(1);
-  const selectedSearchValue = await autocompleteHelper.getSearchInputValue();
```

Comment:
This is very strange indeed… According to the failing assertation, it behaves like the second element is clicked instead of the first one, but I can't see why.

![toto](https://user-images.githubusercontent.com/243653/115377951-6124c000-a1d0-11eb-8e92-0645ddc6cf96.png)